src/driver/postgres/PostgresDriver.ts

```diff
@@ -751,7 +751,15 @@ export class PostgresDriver implements Driver {
                 || tableColumn.isPrimary !== columnMetadata.isPrimary
                 || tableColumn.isNullable !== columnMetadata.isNullable
                 || tableColumn.isUnique !== this.normalizeIsUnique(columnMetadata)
-                || (tableColumn.enum && columnMetadata.enum && !OrmUtils.isArraysEqual(tableColumn.enum, columnMetadata.enum))
+                || (
+                    tableColumn.enum
+                    && columnMetadata.enum
+                    // enums in postgres are always strings
```

Comment:
couldn't we just store enum values as strings in both sides? (e.g. tableColumn.enum and columnMetadata.enum both to be a `string[]` ?)

Reply:
You are half right. `tableColumn.enum` is always `string[]` but for `columnMetadata.enum` we need type `(string|number)[]` because we support numeric and heterogeneous based typescript enums. I updated this PR with typings. Please check it now :)